spec/integration_spec.rb

```diff
@@ -238,6 +240,19 @@ def configure_cocoapods
                             '--swift-build-tool symbolgraph ' \
                               "--build-tool-arguments -I,#{module_path} "
     end
+
+    describe 'Creates docs for Swift project from symbolgraph files' do
```

Comment:
I'd prefer to not change this file, it's quite a lot of overhead to get this working and adding a entire new output set adds change-review burden in the future as well.  I appreciate the effort to add some kind of test for the feature but jazzy doesn't have a way of doing convenient unit-testing.

Reply:
Fair enough, that makes sense. I was hoping that it would be possible to reuse the same before/after states as the main symbolgraph tests, but I think there are some differences in output that would make this impossible. Giving it half-of another shot, but remove this test shortly otherwise.